response/response.go

```diff
@@ -87,17 +87,15 @@ func (res *Response) keep(keepFunc KeepFunc) *Response {
 	return res
 }
 
-// WriteTo is not exported for you, so do not call it directly
-func (res *Response) WriteTo(w http.ResponseWriter) {
+func (res *Response) ServeHTTP(w http.ResponseWriter, r *http.Request) {
```

Comment:
can we pass done channel directly? let `ServeHTTP` won't get too many information.

Reply:
I think this is a good thing since one of our purposes is making sub packages been more independently.

Reply:
With implement `http.Handler`, `response` can be more useful than before

Reply:
I see.